midp/localmsg.js

```diff
@@ -1034,16 +1034,26 @@ MIDP.LocalMsgConnections["nokia.product-info"] = NokiaProductInfoLocalMsgConnect
 
 var localmsgServerWait = null;
 
+// XXX Consolidate the objects we store in this map with those in NativeMap.
+nativeConnectionMap = Object.create(null);
```

Comment:
Is there a reason why you needed a different map or is it just a temporary thing?